src/contactList.py

```diff
@@ -0,0 +1,161 @@
+import sys
+
+
+class Contact:
+
+    def __init__(self, phone, sex, name) -> None:
+        self.setPhone(phone)
+        self.setSex(sex)
+        self.name = name
+
+# check the format of user input for a phone number     
+    def setPhone(self, phone) -> None:
+            try: 
+               intphone = int(phone)
+               self.phone = phone
+            except ValueError:
+                print("Invalid format")
+
+# check the format of user input for sex      
+    def setSex(self, sex) -> None: 
+        upper = sex.upper()
+        if upper == 'F' or upper == 'FEMALE':
+            self.sex = 'Female'
+        elif upper == 'M' or upper == 'MALE':
+            self.sex = 'Male'
+        else: print("Invalid format")
+
+    def print(self):
+        print(self.phone + " " + self.sex + " " + self.name)
+
+class ContactBook: 
+
+    def __init__(self):
+         self.contactList = []
+    
+# add a new contact into the contact list
+    def add_Data(self):
+        
+        print("Please input phone number,sex and name, separated by colon(;): \n ")
+
+        # to format the user's input 
+        txt = input()
```

Comment:
In general, never trust the user input as user can type anything to break your app.